.github/workflows/ci-app_tests.yml

```diff
@@ -0,0 +1,139 @@
+name: CI App Tests
+
+# see: https://help.github.com/en/actions/reference/events-that-trigger-workflows
+on: # Trigger the workflow on push or pull request, but only for the master branch
+  push:
+    branches:
+      - "master"
+  pull_request:
+    paths:
+      - "src/lightning_app/**"
+      - "tests/tests_app/**"
+      - "requirements/app/**"
+      - "setup.py"
```

Comment:
with this we can't set it as required check

Reply:
How should we do it ?